src/NHibernate/Proxy/StaticProxyFactory.cs

```diff
@@ -43,6 +43,11 @@ public override INHibernateProxy GetProxy(object id, ISessionImplementor session
 		}
 
 		public override object GetFieldInterceptionProxy(object instanceToWrap)
```

Comment:
In order to not forget removing it, I think it should be obsoleted, and its base too.

Reply:
Done.